tilecloud_chain/USAGE.rst

```diff
@@ -755,6 +755,8 @@ Environment variables
 - ``TILE_CHUNK_SIZE``: Default to ``1``, the chunk size to process the tiles after the meta tiles.
 - ``TILECLOUD_CHAIN_MAX_OUTPUT_LENGTH``: Default to ``1000``, the maximum number of character of the
   output to be display in the admin interface.
+- ``TILECLOUD_CHAIN_CONCURENT_GET_LEGEND``: Default to ``10``, the number of concurrent request to get the
```

Comment:
default in the code is `20` please decide if you put `20` here or `10` in the code ;P

Reply:
=> 10 everywhere :-)